Create hangman_new.py

ankitranjan9 opened the pull request.

ankitranjan9 committed:
Create hangman_new.py

someshnarwade reviewed (changes requested):
The formatting is very bad, that's why it's showing error while executing.
Please format it properly.

ankitranjan9 committed:
Create hang_man.py

ankitranjan9 committed:
Create words.py

ankitranjan9 committed:
Update README.md

ankitranjan9 commented:
I have done the corrections and moved it to the other_solutions.
thank you

someshnarwade commented on other_solutions/words.py:
```diff
@@ -0,0 +1 @@
+word_list = [    'wares',    'soup',    'mount',    'extend',    'brown',    'expert',    'tired',    'humidity',    'backpack',    'crust',    'dent',    'market',    'knock',    'smite',    'windy',    'coin',    'throw',    'silence',    'bluff',    'downfall',    'climb',    'lying',    'weaver',    'snob',    'kickoff',    'match',    'quaker',    'foreman',    'excite',    'thinking',    'm ...
```
Instead of making a python file use a txt file and like 'word_list.txt'.

someshnarwade commented on hangman_new.py:
```diff
@@ -0,0 +1,11 @@
+import randomfrom words import word_list
```
Delete this file.

someshnarwade commented on other_solutions/README.md:
```diff
@@ -1,3 +1,4 @@
 ## Solutions by other users
 
-Add your solutions here with your github username as python file name.
+you can this program using your command line.
```
Undo these changes

someshnarwade commented on other_solutions/hang_man.py:
```diff
@@ -0,0 +1,146 @@
+import random
```
This works great.

someshnarwade reviewed (changes requested):
I've added comments for the changes required. If you have questions, please ask.

ankitranjan9 committed:
Delete hangman_new.py

ankitranjan9 committed:
Rename words.py to words.txt

ankitranjan9 commented:
I have deleted that file and changed words.py to words.txt. 
everything is done but not able to change the readme I have tried a lot.
please change it by yourself because from tomorrow my exams are going to start.
and please accept my PR this my 4th pr only.

someshnarwade reviewed (approved):
Approved
